usaspending_api/search/delta_models/subaward_search.py

```diff
@@ -253,6 +258,30 @@
             faba.award_id IS NOT NULL
         GROUP BY
             faba.award_id
+    ),
+    state_fips (
+        select
```

Comment:
Can you please capitalize the SQL keywords throughout this file? (`SELECT`, `FROM`, `GROUP`, `BY`, `AS`, etc) to be more consistent with the file. I will add a some documentation around this in our SQL style guide.

Reply:
Ok that'd be great to have this expectation documented.

Reply:
Can you link the style guide?